### DISCUSSION TITLE
Theoritical system requirements

### DISCUSSION THREAD

**Discussion by VirtualPedestrian** [skyline-emu/skyline#194] [General]
What's the theoritical system requirements for the emulator when graphics can be emulated.

**Comment by bylaws** [comment]
An 845 feature level gpu for 2d games and maybe basic 3d, an 855/865 feature level gpu for more advanced 3d games like xenoblade.

This is pure minimum in terms of features, performance is entirely uncertain and you may even need an 888 for games like SMO to run. We will see.

Also > 8gb of ram is recommended
